OpenSSL/SSL.py

```diff
@@ -65,6 +65,7 @@ class _buffer(object):
     pass
 
 OP_SINGLE_DH_USE = _lib.SSL_OP_SINGLE_DH_USE
+OP_SINGLE_ECDH_USE = SSL_OP_SINGLE_ECDH_USE
```

Comment:
Ok, sounds good.

Can you merge master into this branch so that it's up to date and then we
can merge.

On Sat, Sep 5, 2015 at 11:26 AM, Akihiro YAMAZAKI <notifications@github.com>
wrote:

> In OpenSSL/SSL.py
> <https://github.com/pyca/pyopenssl/pull/334#discussion_r38811783>:
>
> > @@ -65,6 +65,7 @@ class _buffer(object):
> >      pass
> >
> >  OP_SINGLE_DH_USE = _lib.SSL_OP_SINGLE_DH_USE
> > +OP_SINGLE_ECDH_USE = SSL_OP_SINGLE_ECDH_USE
>
> oops, sorry. fixed it.
>
> SSL_OP_SINGLE_ECDH_USE was introduced at same time as
> SSL_CTX_set_tmp_ecdh: openssl/openssl@ea26226
> <https://github.com/openssl/openssl/commit/ea262260469e49149cb10b25a87dfd6ad3fbb4ba>
> So, I think conditionally catching exception is unnecessary
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/pyopenssl/pull/334/files#r38811783>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084